tchannel/testing/vcr/cassette.py

```diff
@@ -45,19 +124,23 @@ def _load(self):
         try:
             with open(self.path, 'r') as f:
                 data = f.read()
+                self.existed = True
         except IOError:
             return  # nothing to read
 
         data = yaml.load(data)
-        if 'interactions' in data:
-            if int(data['version']) != VERSION:
-                raise UnsupportedVersionError(
-                    ('Cassette at "%s" is an unsupported version of the '
-                     'format: version %s') % (self.path, str(data['version']))
-                )
-            self._available = [
-                types.Interaction.to_native(i) for i in data['interactions']
-            ]
+        if not (data and 'interactions' in data):
+            return  # file was probably empty
+
+        if int(data['version']) != VERSION:
+            raise UnsupportedVersionError(
+                'Cassette at "%s" is an unsupported version of the '
+                'format: version %s' % (self.path, str(data['version']))
+            )
+
+        self._available = [
```

Comment:
If to-native fails, they probably had an invalid cassette and we want the test to blow up. It's undesirable to silently ignore failures like that during tests.